contracts/token/ERC20/extensions/ERC1363.sol

```diff
@@ -116,8 +116,8 @@ abstract contract ERC1363 is ERC20, ERC165, IERC1363 {
      * Requirements:
      *
      * - The target has code (i.e. is a contract).
-     * - The target `to` must implement the {IERC1363Spender} interface.
-     * - The target should return the {IERC1363Spender} interface id.
+     * - The target `to` must implement {IERC1363Spender-onApprovalReceived}.
```

Comment:
the target should be `spender` not `to`. or better to remove param name in documentation. (also in `_checkOnApprovalReceived`)